### PR TITLE
use the smr macro only together with productname.

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/uyuni-project/uyuni-docs/issues/735

# Description

Fix smr macro usuage.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes https://github.com/uyuni-project/uyuni-docs/issues/735

### CODE REVIEW

File: modules/salt/pages/formula-tftpd.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [[tftpd-formula]]
 = TFTPD Formula
 
-The TFTPD formula is an {smr} formula, used to configure the TFTP service on the branch server.
+The TFTPD formula is an {productname} {smr} formula, used to configure the TFTP service on the branch server.
```
Comment (calancha) [inline modules/salt/pages/formula-tftpd.adoc:4]: Sometimes we write:
`a {productname} {smr}`
or:
`an {productname} {smr}`

The former is suitable for Uyuni, the latter for Suse Manager.
Could we handle this `a` VS `an` systematically?

**Review by calancha — CHANGES_REQUESTED** [review]
How could we handle systematically the `a` VS `an`? I mean,
I see the following things through the source:
`a {productname} {smr}`
`an {productname} {smr}`

this way, we are right 50%  of the time.

**Comment by keichwa** [comment]
Tino Calancha <notifications@github.com> writes:

> this way, we are right 50%  of the time.

Yes, thanks for spotting!  There is no easy way out ;)  Of course,
another indirection layer is always possible: we must run a small
preprocessor step depending on the makefile target.

This idea isn't completly crazy.  We could use a general macro expander
such as 'm4' ;-)

Or ifdef within the adoc files.  I'll let Lana and Joseph decide what we
want to do in this case.

-- 
Karl Eichwalder

**Review by calancha — APPROVED** [review]
I let you decide if you want to merge it and address the `a` VS `an` issue separately,
or if you prefer address both things here.

File: modules/salt/pages/formula-tftpd.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/salt/pages/formula-tftpd.adoc:4]: This happens throughout the docs with {productname}, and I think the adoc ifdef is not fine-grained enough to handle it neatly. I think for retail we could possibly create an additional entity, so we would have one to use with {productname} to create "{productname} for Retail" and one that is just "Retail" for where we want something shorter (or use SMR? is that an approved abbreviation? Might need input from Pau).

**Review by Loquacity — APPROVED** [review]

**Pull request merged**